lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -73,6 +74,26 @@ set_input_value:
       xpath: //input[@id='<input_field_id>']
     op: send_keys <input_value>
     type: input
+# click and check the first item from resource list such as projects, pods, etc.
+click_first_item_from_resource_list_and_check_breadcrumb:
+  action: click_first_item_in_resource_list
```

Comment:
add an action `wait_until_no_loading` before `click_first_item_in_resource_list`

Reply:
ok, added the step and passed.